internal/cmd/gocdk/internal/templates/init.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2019 The Go Cloud Authors
```

Comment:
This should be "The Go Cloud Development Kit Authors". Please fix anywhere you are copy/pasting this from, it keeps coming back like a zombie.